packages/clerk-js/src/ui/components/UserButton/SessionActions.tsx

```diff
@@ -43,12 +43,9 @@ export const SingleSessionActions = (props: SessionActionsProps) => {
         iconElementId={descriptors.userButtonPopoverActionButtonIcon.setId('signOut')}
         icon={SignOut}
         label={localizationKeys('userButton.action__signOut')}
-        variant='secondaryDanger'
```

Comment:
This button needs to be red on hover.
![Screenshot 2024-02-02 at 12 52 33](https://github.com/clerk/javascript/assets/15199353/8211aa42-79fa-4b98-ada2-4e28f7d72ad5)